released/packages/coq-coqprime/coq-coqprime.1.3.0/opam

```diff
@@ -13,7 +13,7 @@ install: [
 ]
 depends: [
   "ocaml"
-  "coq" {>= "8.16~" & < "8.19"}
+  "coq" {>= "8.16~" & < "8.18"}
```

Comment:
```suggestion
  "coq" {>= "8.16~" & < "8.19"}
```
@thery `coq-coqprime.1.3.0` works just fine on Coq 8.18.0, why disallow it?

Reply:
I  want 1.4.0 the version associated with coq platform

Reply:
But in the Platform, the version number for package is typically set manually in a script. So it would suffice if someone sets the version in Platform scripts to `coq-coqprime.1.4.0`, without the need for changing `coq-coqprime.1.3.0`.

Reply:
so is it normal practice in opam to maximized the ranges of dependances?  I thought having overlap between
version is calling for trouble.

Reply:
In my experience, it's common to try to express dependencies "truthfully" in a package, i.e., without regard to other versions. I think it can be an *advantage* to have subsequent versions compatible with the same Coq version, since this means users can migrate from one package version to another seamlessly without changing Coq version.

Reply:
ok I've updated